bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/connection/KNXConnection.java

```diff
@@ -433,6 +440,14 @@ public void updated(Dictionary<String, ?> config) throws ConfigurationException
                 }
             }
 
+            String shouldUseNAT = (String) config.get("useNAT");
+            if (StringUtils.isNotBlank(shouldUseNAT)) {
+                sUseNAT = shouldUseNAT.equalsIgnoreCase("true");
+            }
+            else if (StringUtils.isBlank(shouldUseNAT)) {
+                sLogger.warn("Error when trying to read parameter 'useNAT' from configuration. Parameter was missing, so using default.");
```

Comment:
This is the case I was trying to hadlie in the 'else if':
>    knx:useNAT=

Since the line is not commented out, this would mean the user intended to define a value for the 'useNAT' setting, but for whatever reason, forgot/failed to do so. Even though the setting is optional, it seems like there should be a warning issued for that.

Also, what do you think should be done for the case where the value specified for the setting is not as expected? 
eg:
>    useNAT=falsee